licenses.yaml

```diff
@@ -3184,7 +3184,7 @@ libraries:
 ---
 
 name: Kafka clients
-version: 5.5.12-ccs
+version: 7.0.11-ccs
```

Comment:
looks like this might need to be updated? I'm seeing license check failures:

```
Error: found 3 missing licenses. These licenses are reported, but missing in the registry
druid_module: protobuf-extensions, groupId: io.confluent, artifactId: kafka-schema-registry-client, version: 6.2.12, license: Apache License version 2.0
druid_module: protobuf-extensions, groupId: io.confluent, artifactId: common-utils, version: 6.2.12, license: Apache License version 2.0
druid_module: protobuf-extensions, groupId: org.apache.kafka, artifactId: kafka-clients, version: 6.2.12-ccs, license: Apache License version 2.0
```